kotlin/android/app/build.gradle.kts

```diff
@@ -103,25 +98,20 @@ android {
             )
             isDebuggable = false
 
-            buildConfigField("String", "AUTH_HOST", "\"app.firezone.dev\"")
-            buildConfigField("String", "AUTH_SCHEME", "\"https\"")
-            buildConfigField("Integer", "AUTH_PORT", "443")
-            buildConfigField("String", "CONTROL_PLANE_URL", "\"wss://api.firezone.dev/\"")
+            resValue("string", "app_name", "\"Firezone\"")
 
-            // Docs on filter strings: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html
+            buildConfigField("String", "AUTH_BASE_URL", "\"https://app.firezone.dev\"")
+            buildConfigField("String", "API_URL", "\"wss://api.firezone.dev\"")
             buildConfigField(
                 "String",
-                "CONNLIB_LOG_FILTER_STRING",
+                "LOG_FILTER",
                 "\"connlib_client_android=info,firezone_tunnel=info,connlib_shared=info,connlib_client_shared=info,warn\"",
             )
-
-            resValue("string", "app_name", "\"Firezone\"")
-
             firebaseAppDistribution {
                 serviceCredentialsFile = System.getenv("FIREBASE_CREDENTIALS_PATH")
                 artifactType = "AAB"
                 releaseNotes = "https://github.com/firezone/firezone/releases"
-                groups = "firezone-engineering"
+                groups = "firezone-engineering, firezone-go-to-market"
```

Comment:
@jefferenced If you have an Android device lying around, you should have testing access to the Android Firezone build

Reply:
I don't at the moment, but could always pick up an android device if it's helpful.